acoustid/data/meta.py

```
@@ -4,13 +4,15 @@
 import json
 import logging
 import re
+import time
 import uuid
 from collections import OrderedDict
 from collections.abc import Iterable
 from typing import Any
 
 import six
```

Ruff (0.8.2)

`six` imported but unused

Remove unused import: `six`

(F401)